rules/integrations/aws/initial_access_via_system_manager.toml

```
@@ -24,14 +24,70 @@ interval = "10m"
 language = "kuery"
 license = "Elastic License v2"
 name = "AWS Execution via System Manager"
-note = """## Config
+note = """## Triage and analysis
+
+### Investigating AWS Execution via System Manager
+
+Amazon EC2 Systems Manager is a management service designed to help users automatically collect software inventory, apply
+operating system patches, create system images, and configure Windows and Linux operating systems.
+
+This rule looks for the execution of commands and scripts using System Manager. Note that the actual content of these
+scripts and commands are not included in the event, so analysts must gain visibility using an host-level security product.
```

Technically, the subject here is "content" and so the verb should also be singular ("is not included"), but that feels weird with the plural nouns preceding it. We could make the subject plural instead:
```suggestion
This rule looks for the execution of commands and scripts using System Manager. Note that the actual contents of these
scripts and commands are not included in the event, so analysts must gain visibility using an host-level security product.
```